tests/test_bithack.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from bithacks import bithacks
+
+def test_computeSign():
+    assert(bithacks.computeSign(-42) == -1)
+    assert(bithacks.computeSign(0) == 0)
+    assert(bithacks.computeSign(42) == 1)
+
+def test_detectOppositeSign():

```

Comment:
must match to the function name.

test_detectOppositeSign to test_detectOppositeSign**s**